### ISSUE TITLE
Validation, Test and Training sets.

### ISSUE THREAD

**Issue by moneuron** [8erberg/spatially-embedded-RNN#4]
Hello! Thank you for such a clear wonderful demo notebooks.

I noticed that the code (seRNN_demo.ipynb) defines Training, Validation, and Test sets; however, the Validation set does not appear to be utilized anywhere in the code, with only the Training and Test sets being used. Could you please provide clarification on this?

Thank you.

**Comment by 8erberg** [comment]
Hi moneuron,

thank you for submitting your question. You are right that the validation dataset is not being used in the simple routine highlighted in the notebook. It was left there from an earlier iteration but could simply be removed.

Jascha

**Comment by moneuron** [comment]
I see. Thank you for clarifying!
